src/main/java/org/sagebionetworks/web/client/widget/entity/browse/EntityFinder.java

```diff
@@ -58,6 +65,13 @@ public void configure(boolean showVersions, SelectedHandler<Reference> handler)
 		configure(EntityFilter.ALL, showVersions, handler);
 	}
 	
+	public void configureMultiSelect(boolean showVersions, SelectedHandler<List<Reference>> handler) {
```

Comment:
I don't think showVersions is necessary in this case, but we should validate based on a filter.